x/posts/internal/types/post.go

```diff
@@ -9,18 +9,18 @@ import (
 )
 
 // ---------------
-// --- Post id
+// --- Post newID
 // ---------------
 
-// PostID represents a unique post id
+// PostID represents a unique post newID
 type PostID uint64
 
-// Valid tells if the id can be used safely
+// Valid tells if the newID can be used safely
 func (id PostID) Valid() bool {
 	return id != 0
 }
 
-// Next returns the subsequent id to this one
+// Next returns the subsequent newID to this one
```

Comment:
I think this is a refactoring error

```suggestion
// Next returns the subsequent id to this one
```